.env.example

```
@@ -21,3 +24,10 @@ PUBLIC_CHAT_API_BASE=https://secret-ai-gateway.onrender.com
 CHALLENGE_TOKEN_SECRET=replace-with-a-long-random-secret
 UNLOCK_PUBLIC_KEY=BASE64_PUBLIC_KEY
 UNLOCK_PRIVATE_KEY=BASE64_PRIVATE_KEY
+
+# ===============================
+# IPFS / Pinata (Large Payload Support)
+# ===============================
+PINATA_JWT=replace-with-pinata-jwt
+PINATA_API_URL=https://api.pinata.cloud/pinning/pinJSONToIPFS
+IPFS_GATEWAY_URL=https://gateway.pinata.cloud/ipfs
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)